cms/jinja2/templates/base.html

```diff
@@ -70,3 +115,21 @@
 {% block scripts %}
     <script src="{{ static('js/main.js') }}"></script>
 {% endblock %}
+
+{% block preHeader %}
+    {#
+      NOTE:
+      the "settingsLinkTextURL" setting currently doesn't match what's in the guidelines
+      that's because we're on version 70.0.17 and it has been renamed to "settingsLinkUrl" in 72.0.0
+      See: https://github.com/ONSdigital/design-system/pull/3300
+    #}
+    {{
+    onsCookiesBanner({
+    'lang': "cy" if LANGUAGE_CODE == "cy" else "en",
+    'serviceName': COOKIE_BANNER_SERVICE_NAME,
+    'settingsLinkText': 'Manage settings',
+    'settingsLinkTextURL': MANAGE_COOKIE_SETTINGS_URL,
+    })
+    }}
```

Comment:
```suggestion
    {#
      NOTE:
      the "settingsLinkTextURL" setting currently doesn't match what's in the guidelines
      that's because we're on version 70.0.17 and it has been renamed to "settingsLinkUrl" in 72.0.0
      See: https://github.com/ONSdigital/design-system/pull/3300
    #}
    {# fmt:off #}
    {{
        onsCookiesBanner({
            'lang': "cy" if LANGUAGE_CODE == "cy" else "en",
            'serviceName': COOKIE_BANNER_SERVICE_NAME,
            'settingsLinkText': 'Manage settings',
            'settingsLinkTextURL': MANAGE_COOKIE_SETTINGS_URL,
        })
    }}
    {# fmt:on #}
```

Reply:
Saw this in passing - noting that we will need to update the design system to the latest version (and probably again once the new components are in). Not relevant to this PR but thought I would mention it.